### DISCUSSION TITLE
[Base]   add information from OpenStreetMap on public drinking water

### DISCUSSION THREAD

**Discussion by forwalk-org** [OvertureMaps/data#158] [Discussion]
### Category Feedback

It would be great to add information from OpenStreetMap on public drinking water to produce hiking/cycling maps

https://overpass-turbo.eu/s/1LQM

**Comment by jenningsanderson** [comment]
Thanks for the feedback, @forwalk-org — We can look at adding these water points to the `infrastructure` type in the `base` theme. Stay tuned for the next data release!
